alerts/apache/README.md

```diff
@@ -0,0 +1,39 @@
+# Apache Alerts for Ops Agent
+
+### Notification Channels
+For all alerts, a notification channel needs to be set up or the alert will fire silently.
+
+### User Labels
+User labels can be used for these policies by modifying the userLabels fields of the policies. i.e.
+
+```json
+{ 
+  "userLabels": {
+    "datacenter": "central"
+  }
+}
+```
+
+## High request rate alert
```

Comment:
For consistency with other PRs, could we move the alert policy descriptions above the implementation details of notification channels and user labels? Thanks!